### PR TITLE
feat(graph): support of graph card

### PR DESCRIPTION
**Pull request by acesyde**
#Fixes https://github.com/piitaya/lovelace-mushroom/issues/74
image
image

### CODE REVIEW

**Comment by piitaya** [comment]
Nice 🙂 What's the graph height ? It should be nice to have a `46px` to have the size of the slider, buttons, etc... 
Or we can propose an option to let the user choose the size : compact vs default.

What do you think ?

**Comment by acesyde** [comment]
@piitaya ready to review

**Comment by oziee** [comment]
@acesyde you are a champ!!!! supper work helping to add so much to the work @piitaya has done.. fantastic good you guys

File: src/utils/form/ha-selector.ts [hunk 1]
```diff
@@ -154,8 +154,11 @@ export interface SelectOption {
 
 export interface SelectSelector {
     select: {
+<<<<<<< HEAD
+=======
```
Comment (Kahera) [inline src/utils/form/ha-selector.ts:158]: You've got some conflict markers here

**Comment by alessandroias** [comment]
Wouldn't it be more in line with the rest of the cards if name and value were reversed?
In mushroom usually the `primary_information` is name of the card, while the `secondary_information` is the entity state. I hate to point this out, as I am eager to use this card in my dashboard.

File: docs/cards/graph.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+A graph card allow you to show the actual value and the history.
+
+## Configuration variables
+
+All the options are available in the lovelace editor but you can use `yaml` if you want.
+
+| Name                | Type   | Default     | Description                                    |
+| :------------------ | :----- | :---------- | :--------------------------------------------- |
+| `entity`            | string | Required    | Vacuum entity                                  |
```
Comment (alessandroias) [inline docs/cards/graph.md:16]: You don't want `Vacuum` in here.

**Review by alessandroias — CHANGES_REQUESTED** [review]

**Review by alessandroias — APPROVED** [review]
Looks good to me!

**Comment by alessandroias** [comment]
@piitaya what do you think about this?

**Comment by piitaya** [comment]
I will merge it into a `graph` branch and test it before release 🙂

**Comment by acesyde** [comment]
Yes because it's not up to date 🙂

**Pull request merged**